library/src/otx/backend/native/tools/tile_merge.py

```diff
@@ -508,7 +499,7 @@ def __init__(
     def merge(
         self,
         batch_tile_preds: list[OTXPredBatch],
-        batch_tile_attrs: list[list[dict]],
+        batch_tile_attrs: list[list[TileInfo]],
```

Comment:
Since you are changing name tile_attrs to tile_info, could you change it everywhere for consistency?